src/core/store/modules/lines/lines.selectors.ts

```diff
@@ -230,8 +224,11 @@ const selectUserPositionMetadata = createSelector(
       available: '0',
     };
 
+    //Map over position table
+    //Create a function to pass down position data
+    //Do this in UTIL
```

Comment:
Have these been completed?

Reply:
@Craigson This is not code that I added. I suspect this from @Seroxdesign's PR that I merged into my feature branch in order to test that the query data was making it from Redux state into the frontend.